pkg/toml/util.go

```diff
@@ -0,0 +1,48 @@
+package toml
+
+import (
+	"os"
+	"fmt"
+	"github.com/naoina/toml"
```

Comment:
Good spot, I may have to talk Ioannis through glide. (For Ioannis, that's https://glide.sh/)